### ISSUE TITLE
DB not initialized, please visit setup page to see instructions on how to initialize Trilium

### ISSUE THREAD

**Issue by javait90** [zadam/trilium#309]
can't find Step page , no db info , There's no information

**Comment by kalak-sudo** [comment]
Hi, I keep getting this error on Debian 10 fresh install.

Either with packaged server or Docker:

> 
> Generated sourceId=jtlnARHvnTv1
> App HTTP server starting up at port 8080
> {
>   "appVersion": "0.40.7",
>   "dbVersion": 158,
>   "syncVersion": 14,
>   "buildDate": "2020-03-28T20:58:20+01:00",
>   "buildRevision": "b4b22d9353bdc6ad4d3bab7cdb33bd0b844cc36d",
>   "dataDirectory": "/root/trilium-data",
>   "clipperProtocolVersion": "1.0"
> }
> DB not initialized, please visit setup page to see instructions on how to initialize Trilium.
> 

Any ideas?? Thanks!!

**Comment by zadam** [comment]
You need to open the setup page which is on http://[your-server-host]:8080

**Comment by kalak-sudo** [comment]
Thanks!! Done!! :)

**Comment by ivanvetoshkin** [comment]
Hi, I installed it on my Ubuntu server via Docker, and it gives me the same thing:

> Generated sourceId=TUIYPwwbJKOj
> App HTTP server starting up at port 1999
> {
>   "appVersion": "0.42.3",
>   "dbVersion": 158,
>   "syncVersion": 14,
>   "buildDate": "2020-05-20T08:54:55+02:00",
>   "buildRevision": "04c573e212db06e1dd60c74707e40f6912c85aab",
>   "dataDirectory": "/root/trilium-data",
>   "clipperProtocolVersion": "1.0"
> }
> DB not initialized, please visit setup page - http://[your-server-host]:1999 to see instructions on how to initialize Trilium.

There's no setup page on [my_server_host]:1999. 
As you can see, I even changed the port from 8080 but it doesn't seem to start properly either.
What should I do?

**Comment by zadam** [comment]
> There's no setup page on \[my\_server\_host\]:1999

What do you get when visiting that page?

**Comment by ivanvetoshkin** [comment]
It just loads up infinitely. No, it's not configured in reverse proxy.

**Comment by zadam** [comment]
But then if access isn't configured in reverse proxy then you can't access it. It's difficult to judge from the lack of information but it just sounds like the request are not getting through to the trilium instance ...

**Comment by amadejpapez** [comment]
I also stumbled upon this issue as it was unclear what I have to select there when i created a new server via docker. I kept chosing "I have a server instance..." and getting errors about db and was confused for what to put as the password... but apparently I need to click "I am a new user and create new document for my notes" to go through the first setup. I think this should be more clear somehow.
